### ISSUE TITLE
Register to brightness notifications of the request screen only

### ISSUE THREAD

**Issue by LeoNatan** [michalkonturek/ScreenBrightness#2]
Otherwise, you may hear notifications about brightness events of other screens. iOS allows externals screens to be attached. If these screens report brightness information, your implementation will hear notifications for those screens as well.

**Comment by LeoNatan** [comment]
I see the API already allows the user to specify a screen. So you should be registering to notifications of that screen only.

```swift
    func subscribeToNotifications() {
        let center = self.notificationCenter
        center.addObserver(self,
                           selector: #selector(onScreenBrightnessDidChange),
                           name: UIScreenBrightnessDidChangeNotification,
                           object: screen!)
    }
```

**Comment by michalkonturek** [comment]
Thank you! I will address it shortly.

**Comment by michalkonturek** [comment]
This has been addressed in #3.
